Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -44,9 +50,9 @@ void SFilledGraphWidget::OnImGuiDraw()
 		{
 			XAxisValues.Add(XValue);
 		}
-
-		Algo::Sort(InData2D, [](const FDataPoint& LeftDataPoint, const FDataPoint& RightDataPoint) {return LeftDataPoint.X < RightDataPoint.X; });
-		TArray<float> XAxisValues2D, YAxisValues2D;
+		
```

Comment:
I can't comment on it on the right line! But can you make line 40 use nullptr rather than NULL please.

constexpr bool* CloseButtonVisible = NULL;

NULL is a macro for 0. This mostly works as it'll get implicitly cast to the right type (std::nullptr_t) but there are edge cases where it will break things so the modern way is to use nullptr. (Unless you're calling some API that _explicitly expects_ NULL, there's a few examples of this in the MemFerret codebase where we've had to do this interacting with old windows APIs - I don't know that ImGui::Begin doesn't fall into this category though!).